config.json

```diff
@@ -0,0 +1,27 @@
+{
+  "public": {
+    "default": {
+      "commerce-core-endpoint": "https://www.aemshop.net/graphql",
+      "commerce-endpoint": "https://na1-sandbox.api.commerce.adobe.com/Fwus6kdpvYCmeEdcCX7PZg/graphql",
+      "headers": {
+        "cs": {
+          "Magento-Customer-Group": "b6589fc6ab0dc82cf12099d1c2d40ab994e8410c",
```

Comment:
@sirugh @shiftedreality  I don't think we need "Magento-Customer-Group" in the ACO boilerplate.  If it's needed, let me know what it is for.

Reply:
You can see what uses it in the boilerplate code repo with a search: https://github.com/search?q=repo%3Ahlxsites%2Faem-boilerplate-commerce%20Magento-Customer-Group&type=code

The results suggest it is used for:

- context for search
- context for product-list-page
- "picker" tool
- "segment" tool

For search and plp, this is just part of the search widget's interface. They may or may not be required parameters, and I do not know what will break if we do not include it.